src/index.html

```diff
@@ -9,7 +9,8 @@
     <meta name="apple-mobile-web-app-title" content="KazimirApp">
     <meta name="msapplication-starturl" content="/">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
this is the font that was used in the past, but we self hosted it, not sure how we gonna do it here.